From d496bec5b14df55d49658a283326ad720f974a0a Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Thu, 7 Jun 2018 07:43:15 +0200
Subject: [PATCH 07/15] Disable PCIe-to-PCI bridge device

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <e8b0fa63392143fa6012628221e7b0ab907e4736.1528355911.git.mrezanin@redhat.com>
Patchwork-id: 80594
O-Subject: [RHEL-7.6 qemu-kvm-rhev PATCH 7/7] Disable PCIe-to-PCI bridge device
Bugzilla: 1586357
RH-Acked-by: Thomas Huth <thuth@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

There's new pcie-to-pci device we do not want to support it.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/pci-bridge/Makefile.objs | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/hw/pci-bridge/Makefile.objs b/hw/pci-bridge/Makefile.objs
index 47065f8..68db0da 100644
--- a/hw/pci-bridge/Makefile.objs
+++ b/hw/pci-bridge/Makefile.objs
@@ -1,5 +1,6 @@
 common-obj-y += pci_bridge_dev.o
-common-obj-$(CONFIG_PCIE_PORT) += pcie_root_port.o gen_pcie_root_port.o pcie_pci_bridge.o
+common-obj-$(CONFIG_PCIE_PORT) += pcie_root_port.o gen_pcie_root_port.o 
+#common-obj-$(CONFIG_PCIE_PORT) += pcie_pci_bridge.o
 common-obj-$(CONFIG_PXB) += pci_expander_bridge.o
 common-obj-$(CONFIG_XIO3130) += xio3130_upstream.o xio3130_downstream.o
 common-obj-$(CONFIG_IOH3420) += ioh3420.o
-- 
1.8.3.1

